plugins/basic/nexus-ui-extjs3-plugin/src/main/resources/static/js/Sonatype/utils.js

```diff
@@ -818,7 +817,6 @@ define('Sonatype/utils',['../extjs', 'Nexus/config', 'Nexus/util/Format', 'Sonat
                   method : 'GET',
                   url : Sonatype.config.repos.urls.logout,
                   callback : function(options, success, response) {
-                    ns.clearCookie('JSESSIONID');
```

Comment:
I couldn't figure it out either, but to say it was added pre Nexus 2.1 before multiple shiro upgrades - so concluded it was junk leftover.

Important to note that the call to logout is still made on login failure, and logout now returns NXSESSION=deleteMe, so even though JS clearCookie is not called, effectively behaviour is the same